roles/dosomething.sixpack/tasks/main.yaml

```diff
@@ -12,10 +12,31 @@
     - sixpack
     - gunicorn
 
-- name: Ensures sixpack configuration dir exists
+- name: Ensures sixpack directories exists
   become: yes
   file: path=/etc/sixpack state=directory
+  file:
+    path: /home/dosomething/sixpack
+    state: directory
+    mode: 0755
+    owner: dosomething
+    group: dosomething
 
 - name: Install Sixpack configuration
   become: yes
-  template: src=sixpack.yaml.j2 dest=/etc/sixpack/config.yml owner=root group=root mode=644
\ No newline at end of file
+  template: src=sixpack.yaml.j2 dest=/etc/sixpack/config.yml owner=root group=root mode=644
+
+- name: Configure gunicorn
+  become: yes
+  template: src={{ item }} dest=/{{ item }} owner=root group=root mode=644
```

Comment:
Are you using `/` as pre-pended here instead of full path below since the gunicorn/systemd files might live under other directory prefixes?